src/server/api/controllers/users.controller.js

```diff
@@ -33,26 +59,47 @@ const getUserById = async (id) => {
   return userById;
 };
 
+// SEARCH FOR A USER
 const getUsersByKeyword = async (searchWord) => {
   if (!searchWord) {
     throw new IncorrectEntryError('Use a keyword!', 400);
   }
 
-  const users = await knex('users').where(
-    'fullName',
-    'like',
-    `%${searchWord}%`,
-  );
+  const users = await client.search({
+    index: usersIndex,
+    body: {
+      query: {
+        multi_match: {
+          query: searchWord,
+          type: 'bool_prefix',
+          fields: [
+            'fullName.autocomplete',
+            'fullName._2gram',
+            'fullName._3gram',
```

Comment:
I think this should be:

```suggestion
            'fullName.autocomplete._2gram',
            'fullName.autocomplete._3gram',
```

Reply:
does not seem to make a difference as I test, but changed! ✅

Reply:
I think it makes a difference but it depends on the query and the indexed data so it might not appear to have an impact.